pkg/splunk/enterprise/util.go

```diff
@@ -314,6 +314,41 @@ func appPhaseStatusAsStr(status enterpriseApi.AppPhaseStatusType) string {
 	}
 }
 
+// bundlePushStateAsStr converts the bundle push state enum to corresponding string
+func bundlePushStateAsStr(state enterpriseApi.BundlePushStageType) string {
+	switch state {
+	case enterpriseApi.BundlePushNotStarted:
+		return "Bundle Push Not Started"
```

Comment:
Do we need this state, as we can jump to the Pending, oncee we detect cluster scoped apps?